### DISCUSSION TITLE
Duda sobre comparación de outputs en test cases

### DISCUSSION THREAD

**Discussion by tariztia** [IIC2133-PUC/2024-1#101] [Tarea 1] [answered]
Hola! estaba probando el test case 1 de market y me dice que hay diferencias pero realmente los outputs son iguales.
Este es el error que me aparece:

![image](https://github.com/IIC2133-PUC/2024-1/assets/105675471/cdea6ada-f7c3-4232-8ed0-82ca5fecb0cd)

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, prueba con diff -b
En caso de que no funcione prueba cambiando los \t con espacios o viceversa.

**Reply by tariztia** [reply]
usando diff -b si me funcionó, gracias, pero al momento de evaluar mi entrega no será problema?

**Reply by sebacarrascop** [reply]
no debería serlo, y en el peor de los casos puedes recorregir porque efectivamente tu código está en lo correcto, saludos!
